src/component/term/ParentTermSelector.tsx

```diff
@@ -24,8 +24,24 @@ import { getLocalized } from "../../model/MultilingualString";
 import VocabularyUtils, { IRI } from "../../util/VocabularyUtils";
 import { loadTerms } from "../../action/AsyncActions";
 import HelpIcon from "../misc/HelpIcon";
+import { FaPencilAlt, FaTrashAlt } from "react-icons/fa";
+import TermItState from "../../model/TermItState";
+import Workspace from "../../model/Workspace";
+import TermLink from "./TermLink";
+import BadgeButton from "../misc/BadgeButton";
 import BroaderTypeSelector from "./BroaderTypeSelector";
 
+export const PARENT_ATTRIBUTES = [
+  {
+    attribute: "parentTerms",
+    property: VocabularyUtils.BROADER,
+    labelKey: "term.metadata.parent",
+    selectorLabelKey: "term.metadata.broader",
+    selectorHintKey: "term.metadata.broader.hint",
+  },
+  ...TERM_BROADER_SUBPROPERTIES.slice(),
```

Comment:
AFAIK the `.slice()` is unnecessary here. It creates a new array object as a shallow copy - the content is just references to elements in the original array. So as far as I can tell it doesn't really do anything here.